index.bs

```diff
@@ -257,6 +533,242 @@ A { <code class="vocab">st:hasShapeTreeDecoratorIndex</code> <code>Si</code> }
 	</pre>
 </figure>
 
+Shape Tree Locator {#locator}
+=====================
+
+A <dfn>shape tree locator</dfn> associates a [=managed resource=] with 
+one or more [=shape trees=]. No more than one [=shape tree locator=] may be
+associated with a [=managed resource=]. 
+
+For example, in [[Solid]] a [=shape tree locator=] would be stored in an
+[auxiliary resource](https://solidproject.org/TR/protocol#auxiliary-resources) 
+associated with a given [=managed resource=].
+
+A [=shape tree locator=] includes one or more [=shape tree locations=] via
```

Comment:
I'm tempted to table-ify these definitions to make the list boundaries clear:

## Location properties

property | definition
-- | --
`st:node` | identifies the focus node for [=shape=] validation in the associated [=managed resource=]
`st:shape` | identifies the [=shape=] to which `st:node` must conform
`st:hasShapeTree` | the [=shape tree=] defining this [=shape tree instance=]
`st:hasRootShapeTree` | marks the primary, or parent [=shape tree=] in a physical hierarchy, and is set by the [Plant Operation](#plant-shapetree).
`st:hasShapeTreeInstanceRoot` | marks the primary, or parent [=shape tree instance=] in a physical hierarchy, and is set by the [Plant Operation](#plant-shapetree). <span class="note">@@ define as parm to PLANT?</span>

Note that A [=shape tree root=], and its corresponding [=shape tree instance root=] can be planted within an existing managed hierarchy, alongside or within other [=shape tree roots=] and [=shape tree instance roots=].
Properties like [=Shape tree locations=] in a given [=shape tree locators=] may have different focus nodes.